src/widget/tool/screenshotgrabber.cpp

```diff
@@ -47,6 +47,7 @@ ScreenshotGrabber::ScreenshotGrabber(QWidget* parent)
 
 void ScreenshotGrabber::reInit()
 {
+    delete scene;
     scene = new QGraphicsScene;
     window = new QGraphicsView (scene); // Top-level widget
```

Comment:
This is not part of your changes but could you remove the space in `QGraphicsView (scene)` ?